Sources/PackageDescription/PackageDescriptionSerialization.swift

```diff
@@ -135,7 +144,7 @@ enum Serialization {
         }
 
         let kind: Kind
-        let moduleAliases: [String: String]?
```

Comment:
Unclear why this line is removed?

Reply:
Thanks for the catch. This was due to merging in main. Interestingly though no test failed :D